lab-andrew/entry.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Nice work! I really like that you went the extra mile for getting all previous results. I spotted a few areas that could be improved, but overall really well done.